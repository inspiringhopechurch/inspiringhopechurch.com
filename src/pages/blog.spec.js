import React from "react";
import { shallow } from "enzyme";
import Blog from "./blog";
import data from "../fixtures/data";

describe("Blog", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Blog data={data.data} />);
    expect(wrapper).toMatchSnapshot();
  });
});
