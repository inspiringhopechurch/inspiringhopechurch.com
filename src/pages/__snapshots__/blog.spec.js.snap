// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog should render without crashing 1`] = `
<Fragment>
  <section
    className="hero is-halfheight"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <h1
          className="title has-text-white"
        >
          Latest Blog Posts
        </h1>
      </div>
    </div>
  </section>
  <section
    className="box blog-content"
  >
    <div
      className="columns is-multiline is-centered"
    >
      <BlogItem
        blogDate="01 June, 2020"
        blogExcerpt="The summer is off to a great start! Even though it is definitely different this
year, we have taken advantage of our state opening up some. We've made some more
connections by meeting more neighbors, talking and working with local pastors on
the same mission, and really trying to get involved in the community. Our parks
have opened back up so we've visited the playgrounds a couple of times. We are
planning on doing some community outreaches this next month by providing our
local community worker"
        blogLink="/blog/our-church-planting-journey-volume-6"
        blogTitle="Our Church Planting Journey: Volume 6"
        key="Ghost__Post__5f4aba3168998602af191113"
        onBlogIndex={true}
      />
      <BlogItem
        blogDate="16 May, 2020"
        blogExcerpt="Update...
The last couple of months have been so different than we had expected them to
be. Even in the midst of different, we have found an overwhelming peace with
where God has us and what He is doing. We have clung to our faith and the things
we could control and we prayed and let go and let God work in the ways we
couldn’t. In April we celebrated Ben’s birthday by surprising him with a zoom
call with many of his friends to wish him a happy birthday. Thank you to all of
you who were able to b"
        blogLink="/blog/our-church-planting-journey-volume-5"
        blogTitle="Our Church Planting Journey: Volume 5"
        key="Ghost__Post__5f4ab4bc68998602af1910dc"
        onBlogIndex={true}
      />
    </div>
  </section>
</Fragment>
`;
